toolchest/yaml.py

```diff
@@ -4,6 +4,17 @@
 import yaml
 
 
+class IndentDumper(yaml.Dumper):
+    """ Custom yaml dumper to ensure elements are indented """
+    def increase_indent(self, flow=False, indentless=False):
+        # We can't set indentless from the yaml.dump call and is being set to
```

Comment:
I am sorry, I am having trouble parsing this line. @jpichon if it makes sense to you, we can chalk it up to me being dense today.

Reply:
This is a workaround for pyyaml. If forces indenting off so the only way to get around that is to subclass the Dumper and force it to indent. The sad thing is you can't set this option in yaml.dump.

Reply:
I actually find your comment-explanation much clearer than the code-comment :-) Maybe we could use that??

Reply:
I agree!

Reply:
See if the updated comment helps.